pkg/wallet/wallet_create_action_test.go

```diff
@@ -968,6 +970,59 @@ func (s *WalletTestSuite) TestWalletCreateAction_SendWithAsRetryOfProcessAction(
 }
 
 func (s *WalletTestSuite) TestWalletCreateActionByBobBasedOnAliceCreateAction() {
+	s.Run("alice and bob uses the different storage", func() {
+		t := s.T()
+		const topUpValue = testValueForFunding
+
+		// given:
+		given, cleanup := testabilities.Given(t)
+		defer cleanup()
+
+		// and:
+		aliceWallet := given.Wallet().WithOwnStorage().ForUser(testusers.Alice)
+
+		// and:
+		bobWallet := given.BobWalletWithStorage(s.StorageType)
+
+		// and:
+		_, _ = given.Faucet(aliceWallet).TopUp(topUpValue)
+
+		// when:
+		aliceArgs := fixtures.DefaultWalletCreateActionArgs(t, walletargs.WithLockingScript(script.Script{
+			script.Op3,
+			script.OpEQUAL,
+		}))
```

Comment:
```suggestion
		trivialLockingScript := script.Script{
			script.Op3,
			script.OpEQUAL,
		}
		aliceArgs := fixtures.DefaultWalletCreateActionArgs(t, walletargs.WithLockingScript(trivialLockingScript))
```

And perhaps with some comment why this was utilized, instead of standard P2PKH